File: backend/src/main/java/de/garrafao/phitag/application/annotator/data/command/RemoveAnnotatorCommand.java
package de.garrafao.phitag.application.annotator.data.command;

import lombok.Getter;

@Getter
public class RemoveAnnotatorCommand {

    private final String username;
    private final String project;
    private final String owner;
    private final String annotator;
    private final String entitlement;

    public RemoveAnnotatorCommand(final String username, final String project, final String owner, final String annotator,
                                  final String entitlement) {
        this.username = username;
        this.project = project;
        this.owner = owner;
        this.annotator = annotator;
        this.entitlement = entitlement;
    }
}

File: frontend/lib/model/annotator/command/RemoveAnnotatorCommand.ts
export default class RemoveAnnotatorCommand {
    private readonly username: string;
    private readonly project: string;
    private readonly owner: string;


    constructor(username: string, project: string, owner: string) {
        this.username = username;
        this.project = project;
        this.owner = owner;
    }
    
}